test/unit/components/reveal.test.js

```diff
@@ -0,0 +1,45 @@
+const expectComponent = require('../component-helper').expectComponent;
+
+describe('Reveal component', () => {
+  it('should render', () => {
+    expectComponent(
+      'reveal',
+      {
+        variant: 'block',
+        summary: 'Summary text',
+        children: [],
+      },
+      `<details class="">
+        <summary data-analytics="summary">
+          <span class="details__summary">
+      Summary text
+    </span>
```

Comment:
Ah. Yes, that's because the pretty formatter errors when indented. Seems to not _beautify_ children of spans. 

Maybe your suggestion of testing the components minified might be more robust but then it's hard to see which line is affecting that diff when tests fail.